Issues running GTN website locally on Mac

Hi, I was having issues with `make serve` and ended up doing a fresh clone of this repo. But when running `make create-env` (as [here](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/running-jekyll/tutorial.html)) I get the error
```
Collecting package metadata (repodata.json): done
Solving environment: failed

ResolvePackageNotFound: 
  - gxx=9
  - gcc=9

make: *** [create-env] Error 1
```

I'm guessing as there's no gcc9 (or g++9) for Mac in conda here https://anaconda.org/conda-forge/gcc ? 

I'll try using GitPod in the meantime but does anyone know if/how I can get past this error?

--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,5 @@
 # Settings
+UNAME := $(shell uname)
 JEKYLL=jekyll
 PORT?=4000
 HOST?=0.0.0.0
@@ -20,6 +21,12 @@ ifeq ($(shell uname -s),Darwin)
 	MINICONDA_URL=https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
 endif
 
+ifeq ($(UNAME),Darwin)
+	ENV_FILE=environment-osx.yml
+else
+	ENV_FILE=environment.yml
+endif
+
 CONDA=$(shell which conda)
 ifeq ($(CONDA),)
 	CONDA=${HOME}/miniconda3/bin/conda
@@ -34,9 +41,9 @@ install-conda: ## install Miniconda
 
 create-env: ## create conda environment
 	if ${CONDA} env list | grep '^${CONDA_ENV}'; then \
-	    ${CONDA} env update -f environment.yml; \
+	    ${CONDA} env update -f ${ENV_FILE}; \
 	else \
-	    ${CONDA} env create -f environment.yml; \
+	    ${CONDA} env create -f ${ENV_FILE}; \
 	fi
 .PHONY: create-env
 

--- a/Makefile
+++ b/Makefile
@@ -257,6 +257,7 @@ rebuild-search-index: ## Rebuild search index
 	node bin/lunr-index.js > search.json
 
 api/swagger.json: metadata/swagger.yaml
+	$(ACTIVATE_ENV) && \
 	cat metadata/swagger.yaml | python bin/yaml2json.py > api/swagger.json
 
 clean: ## clean up junk files

--- a/environment-osx.yml
+++ b/environment-osx.yml
@@ -1,0 +1,28 @@
+---
+name: galaxy_training_material
+channels:
+  - conda-forge
+  - bioconda
+dependencies:
+  - clang
+  - gmp>=6.1.2
+  - jemalloc>=5.0.1
+  - libiconv>=1.15
+  - nodejs>=9.11.1
+  - openssl>=1.0.2o
+  - pandas>=0.22.0
+  - pip
+  - pip:
+    - pathspec==0.5.6
+    - oyaml
+  - planemo>=0.55.0
+  - readline
+  - requests>=2.18.4
+  - ruby=2.7
+  - libffi
+  - yaml>=0.1.7
+  - yamllint>=1.11.0
+  - zlib
+  - libxml2
+  - ephemeris
+  - pkg-config

